shotover-proxy/examples/redis-cluster-ports-rewrite/docker-compose.yml

```diff
@@ -42,16 +75,19 @@ services:
       - redis-node-0
       - redis-node-1
       - redis-node-2
+      - redis-node-3
+      - redis-node-4
+      - redis-node-5
     environment:
       - 'ALLOW_EMPTY_PASSWORD=yes'
-      - 'REDIS_CLUSTER_REPLICAS=0'
-      - 'REDIS_NODES=redis-node-0 redis-node-1 redis-node-2'
+      - 'REDIS_CLUSTER_REPLICAS=1'
```

Comment:
which one is the replica?

Reply:
Its the number of replicas for every master that the cluster has. We end up with 3 masters and 3 replicas: 

![image](https://user-images.githubusercontent.com/39339036/146288070-fb5ca4ef-cd3c-41d2-aa4b-61f7664d318b.png)